README.md

```diff
@@ -73,7 +84,7 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+[x] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
 [] Synchronous code runs all at the same time, asynchronous code runs completely randomly
 [] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
```

Comment:
This one is actually the correct answer. e.g. `setTimeout` may run at an even pace, but the order will not be what appears in the code:

```js
setTimeout(function() {
   alert("this appears second");
}, 1000);

alert("this appears first");
```